internal/process/provisioning/create_runtime_resource_step_test.go

```diff
@@ -389,6 +389,141 @@ func TestCreateRuntimeResourceStep_HandleNotNilOIDCWithoutListOrObject(t *testin
 	assert.Equal(t, expectedOIDCConfig, (*runtime.Spec.Shoot.Kubernetes.KubeAPIServer.AdditionalOidcConfig)[0])
 }
 
+func TestCreateRuntimeResourceStep_HandleOIDCWithJwks(t *testing.T) {
```

Comment:
Additional test cases:

- empty param (empty list)
- dash/minus param
- invalid (not base64 param)